src/dswx_sar/dswx_runconfig.py

```diff
@@ -19,6 +19,10 @@
 
 # Potential polarization scenarios for DSWx-S1
 # NOTE: DO NOT CHANGE THE ORDER of the items in the dictionary below.
+# There are more modes such as 'MIX_DUAL_V_SINGLE_V' and 
+# 'MIX_DUAL_H_SINGLE_H' describing the dual_polarization and
+# single polarization with same polarization. This mode will be 
+# deteremine by counting the available bursts. 
 DSWX_S1_POL_DICT = {
     'CO_POL': ['HH', 'VV'],
```

Comment:
Yes. I excluded two modes in the dictionary here because both may cause complicated issues in finding the correct mode. For example, `DV_POL` has the same elements with `MIX_DUAL_V_SINGLE_V_POL`. This mode will be determined by counting the number of Geotiffs having the co- and cross-pol.